README.md

```diff
@@ -1,55 +1,86 @@
-# IBM Blockchain Extension for VSCode
+# IBM Blockchain Platform Extension for VSCode
 <!---Installing instructions
 --->
 <!---Short description of what the extension allows the user to do and key features in bullet points below 
 --->
-The IBM Blockchain extension allows a user to write and test chaincode projects in Visual Studio Code:
-* Generate skeleton chaincode projects
-* Adds a Fabric view to connect and interact with Fabric Blockchain networks
+The IBM Blockchain Platform extension allows a user to easily write and test smart contract projects in Visual Studio Code:
+* Generate skeleton smart contract projects
+* Adds a IBM Blockchain Platform view to connect and interact with Fabric Blockchain networks
 
+## Installation
 
-## Connecting to your own Hyperledger Fabric instance
+**[Install the IBM Blockchain Platform extension here](vscode:extension/IBM.ibm-blockchain-platform)**
 
-Using this extension, you can connect to a pre-configured local instance of Hyperledger Fabric, or you can connect to your own Hyperledger Fabric instance. If you choose to connect to your own Fabric instance, it must be Fabric 1.3.0 or later.
 
-**When using a local Hyperledger Fabric instance the extension will automatically pull and tag the required Docker images.**
+Or visit the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.ibm-blockchain-platform) for more details
 
-To connect to your own Hyperledger Fabric instance within the extension, you must first pull and tag the Fabric 1.3.0 Docker images by running the following commands:
+## Requirements
 
-```
-docker pull nexus3.hyperledger.org:10001/hyperledger/fabric-ca:amd64-1.3.0-stable
-docker tag nexus3.hyperledger.org:10001/hyperledger/fabric-ca:amd64-1.3.0-stable hyperledger/fabric-ca
+You will need the following installed in order to start a local Hyperledger Fabric runtime from within the extension:
+- Node v8.x or greater
+- npm v5.x or greater
 
-docker pull nexus3.hyperledger.org:10001/hyperledger/fabric-orderer:amd64-1.3.0-stable
-docker tag nexus3.hyperledger.org:10001/hyperledger/fabric-orderer:amd64-1.3.0-stable hyperledger/fabric-orderer
+*(Note: We recommend installing Node and npm using the ['node version manager'](https://github.com/creationix/nvm))*
```

Comment:
We cannot legally recommend anything, I know this rule stands on IBM hosted content, presume it also exists for pages like this created by IBM. (This is because once we recommended something to a customer, then they did it and it went badly and they sued us.)

Honestly I'd probably just delete this line unless we _really_ care about people using nvm.